package.json

```
@@ -8,6 +8,7 @@
     "build": "node scripts/build.js",
     "lint": "./node_modules/.bin/tslint --config tslint.json --project tsconfig.json",
     "lint:ci": "./node_modules/.bin/tslint --config tslint.ci.json --project tsconfig.json -t junit > lint-results.xml",
+    "eslint": "./node_modules/.bin/eslint --ext .ts src",
```

I would suggest we add `--ext .tsx` as well so we get more linting coverage.

Included extensions | Errors | Warnings
--- | --- | ---
`.ts` only | 153 | 53
`.ts` + `.tsx` | 737 | 192

```suggestion
    "eslint": "./node_modules/.bin/eslint --ext .ts --ext .tsx src",
```

---

fixed. errors more than fill my terminal